packages/vue-components/src/__tests__/Breadcrumb.spec.js

```diff
@@ -4,7 +4,7 @@ import Breadcrumb from '../Breadcrumb.vue';
 
 const SITENAV_START = `
 <nav id="site-nav">
-<div class="nav-component slim-scroll>
+<div class="nav-component" slim-scroll>
```

Comment:
```suggestion
<div class="nav-component slim-scroll">
```
i think slim-scroll is a classname